### PR TITLE
(WIP) Climbing Adjustments

### PR DESCRIPTION
**Pull request by Alzter**
I adjusted Tux's climbing speed to be faster, and made it where pressing action doesn't take you off climbables since that seems kind of impractical.

I wanted to make it where Tux doesn't fall off climbables when he reaches the edge of the area, but he just isn't allowed to move outside them, but I wasn't sure how to code it. If anyone else could that'd be appreciated!

### CODE REVIEW

File: src/object/player.cpp [hunk 1]
```diff
@@ -774,7 +774,7 @@ Player::handle_horizontal_input()
     }
     else {
       // give Tux tighter air control
-      ax *= 2.f;
+      ax *= 1.5f;
```
Comment (Zwatotem) [inline src/object/player.cpp:777]: This seems to be unrelated to climbing.

Comment (Alzter) [reply src/object/player.cpp:777]: > This seems to be unrelated to climbing.

Oops, I was messing with Tux's velocity and forgot to reset it.

**Comment by Alasdairbugs** [comment]
Action should be allowed to eject tux from a ladder... otherwise how else is he going to get off the ladder if he's climbing halfway through? move over to the side until he falls off??? No, that's way too slow.

If you're going to remove action taking tux off the ladder I suggest replacing it with down, since you're never going to be ground-pounding or ducking on a ladder, it makes sense.

**Comment by Zwatotem** [comment]
My plan was to make `JUMP+DOWN` work as `ACTION`, so you still can let off without turning in any direction. Additionally `JUMP + any direction combination` (including diagonals) should send Tux jumping off the ladder in that direction.

**Comment by Alzter** [comment]
> If you're going to remove action taking tux off the ladder I suggest replacing it with down, since you're never going to be ground-pounding or ducking on a ladder, it makes sense.

But isn't pressing down needed to climb down the ladder?

**Comment by Ordoviz** [comment]
> But isn't pressing down needed to climb down the ladder?

It would be <kbd>Down</kbd> + <kbd>Jump</kbd>.

**Comment by Alzter** [comment]
> > But isn't pressing down needed to climb down the ladder?
> 
> It would be Down + Jump.

I know, but that wasn't what they said. I don't understand why it can't just be jump

**Comment by Semphriss** [comment]
I can look into keeping Tux on the ladders, if help is still needed about it.

**Comment by Guih48** [comment]
Yes, I think too: the current climbable "climb-off" behaviour is wery irritating. 

At the edges of climbables, Tux must stop, and don't take off without direct order (jump (or action) buttons) but the "climb-off" isn't fully a bad behaviour, it can be good with just one improvement: 
You automatically stop and the edges, and if you want to climb-off, you just release and press again the left, the right or the down key (not the up, because it's also could make irritating situations) (of course, if there is a safe ground at the climbable's bottom, if you touch the bottom of the climbable, you automatically "climb-off" to the ground).

For take-off from the climbable I agree with @Zwatotem 's idea(if we can't agree with the current action key method):

> My plan was to make `JUMP+DOWN` work as `ACTION`, so you still can let off without turning in any direction. Additionally `JUMP + any direction combination` (including diagonals) should send Tux jumping off the ladder in that direction.

I think, this is the best, if we wouldn't use action for jumping off (and ground-pounding isn't a problem, because it needs falling time to gain power, and if you don't want to gtound-pound, you just release the down key.

**Comment by weluvgoatz** [comment]
Closing because these changes seem to not be planned or worked on, and even if they were, would likely be included in a much more expansive PR anyway.
